docs/src/modules/utils/postProcessImport.ts

```diff
@@ -1,7 +1,6 @@
-// eslint-disable-next-line import/no-relative-packages
-import pickersPackageJson from '../../../../packages/x-date-pickers/package.json';
+import adapterDependencies from './adapter-dependencies.json';
```

Comment:
The previous solution was sketchy, but "worked".
Sadly, with the introduction of `catalog:`, static imports are no longer enough as we need to resolve or use resolved dep versions.

If you have better ideas on how to handle this case, I'm all ears, but let's not forget that this runs in Next.js docs runtime/browser, not Node. 🙈